From f609bea45d33e2feeb36e107f06e85b621732b4f Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Tue, 12 Apr 2016 16:28:11 +0300
Subject: [PATCH 1034/1240] net: mvpp2x: Remove unused swf_size variable

- Remoze unused swf_size int from mv_pp2x_tx_queue struct

Change-Id: If3a27a58ffe70d4834081a519a4cab9b33c6446f
Reviewed-on: http://vgitil04.il.marvell.com:8080/28972
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 drivers/net/mv_pp2x.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/net/mv_pp2x.h b/drivers/net/mv_pp2x.h
index d8bb1d3..eb7aa16 100644
--- a/drivers/net/mv_pp2x.h
+++ b/drivers/net/mv_pp2x.h
@@ -2979,9 +2979,6 @@ struct mv_pp2x_tx_queue {
 	/* Number of Tx DMA descriptors in the descriptor ring */
 	int size;
 
-	/* Number of Tx DMA descriptors to be used in software forwarding */
-	int swf_size;
-
 	/* Number of currently used Tx DMA descriptor in the
 	 * descriptor ring
 	 */
-- 
1.9.1

